logging/cluster-logging-loki.adoc

```diff
@@ -26,6 +26,8 @@ include::modules/logging-loki-cli-install.adoc[leveloffset=+1]
 
 include::modules/logging-clo-cli-install.adoc[leveloffset=+1]
 
+include::modules/configuring-log-storage-cr.adoc[leveloffset=+1]
```

Comment:
More of a nice-to-have, but would adding the rendered headings as file comments would help for easier preview identification.  Example:

// Troubleshooting Loki rate limit errors